ci/prow/e2e-crd-variable

```diff
@@ -3,11 +3,10 @@
 set -euxo pipefail
 
 echo "Get first node..."
-NODE=$(oc get nodes -o jsonpath='{.items..metadata.name}' | awk {'print $1'})
-
-source ci/prow/check-kmm-kmod
+oc get nodes -o jsonpath='{.items..metadata.name}' | awk {'print $1'} > nodename
```

Comment:
I think that if you `export NODE=...` instead of `NODE=...` it will work.

Reply:
You are right. I'll remove the output to the temp file.